File: launchdarkly/python/src/PatchMemberExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    patch_operation_1 = models.PatchOperation(
        op="add",
        path="/role",
    )

    patch_operation = [
        patch_operation_1,
    ]

    try:
        response = api.AccountMembersApi(api_client).patch_member(
            id=None,
            patch_operation=patch_operation,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling AccountMembers#patch_member: %s\n" % e)

File: launchdarkly/ruby/src/PatchMemberExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

patch_operation_1 = LaunchDarklyClient::PatchOperation.new
patch_operation_1.op = "add"
patch_operation_1.path = "/role"

patch_operation = [
    patch_operation_1,
]

begin
    response = LaunchDarklyClient::AccountMembersApi.new.patch_member(
        "id_string", # id
        patch_operation,
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling AccountMembersApi#patch_member: #{e}"
end
